generators/uscore-r4/generator.rb

```diff
@@ -116,7 +129,7 @@ def create_search_test(sequence, search_param)
 
         @#{sequence[:resource].downcase} = reply.try(:resource).try(:entry).try(:first).try(:resource)
         @#{sequence[:resource].downcase}_ary = reply&.resource&.entry&.map { |entry| entry&.resource }
-        save_resource_ids_in_bundle(versioned_resource_class('#{sequence[:resource]}'), reply)
+        save_resource_ids_in_bundle(versioned_resource_class('#{sequence[:resource]}'), reply#{', ' + validation_profile_uri(sequence) if validation_profile_uri(sequence)})
```

Comment:
I think this would be nicer if this logic happened outside of the string. Something like:
```
save_resource_ids_in_bundle_arguments = [
  "versioned_resource_class('#{sequence[:resource]}')",
  'reply',
  validation_profile_uri(sequence)
].compact.join(', ')
```

Reply:
fixed